demisto_sdk/commands/validate/validators/PB_validators/PB123_is_conditional_task_has_unhandled_reply_options.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsAskConditionHasUnhandledReplyOptionsValidator(BaseValidator[ContentTypes]):
+    error_code = "PB123"
+    description = "Checks whether an ask conditional has unhandled reply options."
+    rationale = "Checks whether an ask conditional has unhandled reply options."
+    error_message = (
+        "Playbook conditional task with id:{task_id} has an unhandled "
+        "condition: {condition}."
+    )
+    related_field = "conditions"
+    is_auto_fixable = False
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        """Checks whether a builtin conditional task branches are handled properly
+        Args:
+            - content_items (Iterable[ContentTypes]): The content items to check.
+        Return:
+            - List[ValidationResult]. List of ValidationResults objects.
+        """
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            invalid_tasks = self.unhandled_conditions(content_item)
+
+            for task_id, unhandled_options in invalid_tasks.items():
```

Comment:
break the nested for

Reply:
Seems like there is a dependency down the road which prevents breaking the nested for loop